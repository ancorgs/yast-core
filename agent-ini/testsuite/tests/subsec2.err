[yast2-agent-ini] IniParser.cc(parse_helper):XXX 3: Section s2 started but section s1 is not finished
[yast2-agent-ini] IniParser.cc(parse_helper):XXX 6: Val2: values at the top level not allowed.
[yast2-agent-ini] IniParser.cc(parse_helper):XXX 7: Nothing to close.
