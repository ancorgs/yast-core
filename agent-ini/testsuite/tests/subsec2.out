(["s1", "s2"])
Section s1
Val1=v1
End s1
Section s2
Val1=v3
End s2
